pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -182,35 +268,70 @@ var _ backupManifestFileIterator = &sstFileIterator{}
 // the first level are instead used to extend the current rightmost span in
 // if its current data size plus that of the new span is less than the target
 // size.
-func makeSimpleImportSpans(
+func makeStreamingImportSpans(
```

Comment:
nit: this name seems liable to cause a little confusion given we have a feature that is sometimes called "streaming" and some package names, etc that are DR adjacent. maybe makeAndSendImportSpans or something?

Reply:
Done, changed to `generateAndSendImportSpans`, also added a comment describing how the covering is created.